bedrock/firefox/forms.py

```diff
@@ -10,8 +10,6 @@
 
 from lib.l10n_utils.dotlang import _lazy as _
 
-LANG_FILES = ['firefox/whatsnew-fx37']
-
```

Comment:
That page was done only for a [very small set of locales](https://github.com/mozilla-l10n/langchecker/commit/b732eca001f5a2405e2c93f1f106315bc84801f5#diff-913ec76ed4663fa42b51ed974ca60902L922) (that's one of the few I never dealt with).

Is this message actively used in any page?